Source/dom/layers/Layer.js

```diff
@@ -252,13 +252,13 @@ Layer.define('selected', {
 
 Layer.define('flow', {
   get() {
-    return wrapObject(this._object.flow())
+    return Flow.fromNative(this._object.flow())
   },
   set(_flow) {
     if (this.isImmutable()) {
       return
     }
-    const flow = Flow.from(_flow)
+    const flow = _flow ? Flow.from(_flow) : null
```

Comment:
Wouldn't `flow.sketchObject` in the following line throw in error if `flow` turns out to be `null`?

Further, is there a particular reason to use `null` here, especially if the tests expects `undefined`?

Reply:
the reason to use `null` is that it gets translated to `nil` which is what obj-c expects. `undefined` is translated to `MOUndefined` because `undefined` just doesn't exist in Obj-C